CellProfiler.spec

```diff
@@ -67,11 +63,11 @@ exe = EXE(
     pyz,
     a.scripts,
     # options,
-    console=False,
+    console=True,
     debug=False,
     exclude_binaries=True,
-    icon=os.path.join("cellprofiler", "data", "CellProfilerIcon.ico"),
-    name='CellProfiler-App',
+    icon=os.path.join("cellprofiler", "data", "images", "CellProfilerIcon.ico"),
+    name="cp",
```

Comment:
What's the purpose of the `name` field? 

`"cp"` seems a little terse. Is `"cellprofiler"` more appropriate?